geoportal/geoportailv3_geoportal/static-ngeo/js/LayerPermalinkManager.js

```diff
@@ -192,7 +192,25 @@ exports.prototype.onLayerUpdate_ = function(layers) {
   var bgLabel = 'blank';
   if (bgLayer) {
     bgLabel = bgLayer.get('label');
+
+  // If there is a expert mvt style in the localstorage but no parameter in the url
+    const itemKey = 'remoteIdForStyle_' + bgLabel;
```

Comment:
maybe this pattern should not be repeated here but imported… not blocking btw.

Reply:
I did not imported it from MvtStylingService as I would need to import the whole service into the LayerPermalinkManager just for a single function to use... So I just copied the needed line to get the correct label here.